### ISSUE TITLE
bspwmrc has to be loaded twice for changes to take effect 

### ISSUE THREAD

**Issue by kbrgl** [baskerville/bspwm#501]
I've been having to load my bspwmrc manually for some changes to take effect. When loaded by bspwm, all external programs are started (feh, panel, compton, etc.) but a couple of bspc commands appear to have no effect, specifically: `border_width`, `window_gap`, `top_padding`, `left_padding`, `right_padding` and `bottom_padding` (non-exhaustive). To make these settings take effect, I have to load the bspwmrc manually.

```sh
#!/bin/sh

bspc monitor -d 1 2 3 4 5 6 7 8 9 0

bspc config border_width        4
bspc config window_gap          20
bspc config top_padding         50
bspc config left_padding        15
bspc config right_padding       15
bspc config bottom_padding      15

bspc config split_ratio                       0.5
bspc config focus_follows_mouse               true
bspc config focus_by_distance                 false
bspc config borderless_monocle                true
bspc config gapless_monocle                   true

bspc config focused_border_color "#eeeeee"
bspc config active_border_color "#eeeeee"
bspc config normal_border_color "#212121"

feh --bg-fill /home/kabir/Pictures/bg.jpg

compton &
/home/kabir/.local/bin/panel &
```
The file is marked as executable.

**Comment by kbrgl** [comment]
Ah, apparently this has been fixed (#186). Closing.

**Comment by baskerville** [comment]
The underlying problem was fixed in ebad909.
